.github/workflows/continuous-e2e-different-node-speed-and-reliability-test.yaml

```diff
@@ -17,7 +17,6 @@ jobs:
     secrets: inherit
```

Comment:
since you are changing this file already, do you mind appropriately naming this test, it is not starting with run-forge-.. like all others (sorry)

run-different-node-speed-and-reliability-test => run-forge-different-node-speed-and-reliability-test: